### PR TITLE
App action interface for leapfrog 7334

### PR DESCRIPTION
**Pull request by asteyer**

### CODE REVIEW

File: packages/tempus/src/Tempus_StepperLeapfrog_decl.hpp [hunk 1]
```diff
@@ -12,7 +12,11 @@
 #include "Tempus_config.hpp"
 #include "Tempus_StepperExplicit.hpp"
 #include "Tempus_StepperObserverComposite.hpp"
```
Comment (ccober6) [inline packages/tempus/src/Tempus_StepperLeapfrog_decl.hpp:14]: You should be able to move #include "Tempus_StepperObserverComposite.hpp" inside TEMPUS_HIDE_DEPRECATED_CODE.

File: packages/tempus/src/Tempus_StepperLeapfrog_impl.hpp [hunk 2]
```diff
@@ -42,15 +45,41 @@ StepperLeapfrog<Scalar>::StepperLeapfrog(
...
+  std::string ICConsistency,
+  bool ICConsistencyCheck,
+  const Teuchos::RCP<StepperLeapfrogAppAction<Scalar> >& stepperLFAppAction)
+  {
+    this->setStepperType(        "Leapfrog");
+    this->setUseFSAL(            useFSAL);
+    this->setICConsistency(      ICConsistency);
+    this->setICConsistencyCheck( ICConsistencyCheck);
 
+    this->setObserver();
```
Comment (ccober6) [inline packages/tempus/src/Tempus_StepperLeapfrog_impl.hpp:71]: Need to put TEMPUS_HIDE_DEPRECATED_CODE around the setObserver().

**Comment by ccober6** [comment]
Looks good.  Just a couple of minor changes and it should be ready to go!

**Comment by asteyer** [comment]
@ccober6 This should be ready for a final review once it passes the tests.

**Comment by asteyer** [comment]
@ccober6 This is ready for your review.

**Review by ccober6 — APPROVED** [review]
Looks good!

**Pull request merged**